ENTERPRISE_FEATURES.md

```diff
@@ -0,0 +1,456 @@
+# Premium Enterprise Features
```

Comment:
I would suggest adding a table of contents (or at the least a list of the features included in this file) to make it easier to navigate if for example a client was only looking for the cross device URL to avoid them needing to scroll all the way through the content or thinking they are on the wrong page.

Reply:
Good point. Have added a table of contents in based on the features listed.